src/actions/actionsFactory.ts

```diff
@@ -1,36 +1,38 @@
 import { Uri } from "vscode";
-import { IAction, ICommandAction, ActionType, IFileAction, ActionJsonKey, ISnippetAction } from './interfaces';
+import { ICommandAction, IFileAction, ISnippetAction, ActionType, BasAction } from '@sap-devx/app-studio-toolkit-types';
 import { CommandAction, FileAction, SnippetAction } from './impl';
+import { COMMAND, SNIPPET, FILE, URI } from '../constants';
 
 const getNameProp = (fromSettings: boolean): string => {
-    return fromSettings ? "name" : ActionJsonKey.CommandName;
+    return fromSettings ? "name" : "commandName";
 };
 const getParamsProp = (fromSettings: boolean): string => {
-    return fromSettings ? "params" : ActionJsonKey.CommandParams;
+    return fromSettings ? "params" : "commandParams";
 };
 
 export class ActionsFactory {
```

Comment:
The action factory is one of the main flows that should be tested (manually)
As this is one of the only runtime changes to the production code